Source/JavaScriptCore/wasm/WasmTypeDefinition.cpp

```diff
@@ -117,16 +153,42 @@ static unsigned computeStructTypeHash(size_t fieldCount, const StructField* fiel
     return accumulator;
 }
 
+static unsigned computeRecursionGroupHash(size_t typeCount, const TypeIndex* types)
+{
+    unsigned accumulator = 0x9cfb89bb;
```

Comment:
How do we pick these?

Reply:
I've been picking by generating a random hex number, but if there is a more principled way to do it I would be happy to change this.